.github/workflows/check-patch.yaml

```diff
@@ -0,0 +1,19 @@
+name: Lint Python
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install requirements
+      run: bash ./scripts/ci/install.sh
+    - name: Check Patch
+      run: ./scripts/ci/check_patch.sh
```

Comment:
Looks like we can now remove it from travis?

Reply:
@efiop, mentioned here: https://github.com/iterative/dvc/pull/4092#issuecomment-648300217

Reply:
Oops, missed that one. Ok, let's keep it for now.